### ISSUE TITLE
EOL Spacing

### ISSUE THREAD

**Issue by 5u623l20** [jdkato/txtlint#501]
I am trying to write a style to error on End of Line spaces. So far I have done this:

```yml
extends: existence
message: "End of Line should not have spaces."
level: error
nonword: true
tokens:
  - '[.]\s+$'
 ```

But it doesn't seem to catch the end of line spaces.

**Comment by 5u623l20** [comment]
Fixed with:
```yml
---
extends: existence
message: "End of Line should not have spaces."
level: warning
scope: raw
raw:
  - ' +\n'
```
